lmdeploy/model.py

```diff
@@ -575,6 +575,73 @@ def messages2prompt(self, messages, sequence_start=True):
         return super().messages2prompt(messages, sequence_start)
 
 
+@MODELS.register_module(name='solar')
+class SOLAR(BaseModel):
+    """Chat template of SOLAR model.
+
+    `https://huggingface.co/upstage/SOLAR-0-70b-16bit`
+    """
+
+    def __init__(self,
+                 b_sys='### System:\n',
```

Comment:
May change the argument name in another PR. Try to keep them consistent with other chat template, like `system`, `user`, `asssitant`